README.rst

```diff
@@ -42,16 +63,67 @@ Add the AutoExtract downloader middleware in the settings file::
 
 Note that this should be the last downloader middleware to be executed.
 
+The providers
+-------------
 
-Usage
-=====
+Another way of consuming AutoExtract API is using the Page Objects pattern
+proposed by the `web-poet`_ library and implemented by `scrapy-poet`_.
 
-The middleware is opt-in and can be explicitly enabled per request,
-with the ``{'autoextract': {'enabled': True}}`` request meta.
-All the options below can be set either in the project settings file,
-or just for specific spiders, in the ``custom_settings`` dict.
+Page Objects their returned Items are defined by the `autoextract-poet`_
+library.
+
+Within the spider, consuming the AutoExtract result is as easy as::
+
+    from autoextract_poet.page_inputs import AutoExtractArticleData
+
+    def parse(self, response, article: AutoExtractArticleData):
+        yield article.to_item()
+
+Note that on the example above, we're going to perform two requests:
+
+* one goes through Scrapy (it might use Crawlera, Splash or no proxy at all, depending on your configuration)
+* another goes through AutoExtract API using `scrapinghub-autoextract`_
+
+If you don't need the additional request going through Scrapy,
+you can annotate the response argument of your callback with the DummyResponse type.
```

Comment:
``DummyResponse``

Reply:
Sorry, what's supposed to be changed here?

Reply:
DummyResponse is not formatted properly 
Forgot that it would be formatted in the comment :sweat_smile: